splunk_add_on_ucc_framework/__init__.py

```diff
@@ -365,9 +365,17 @@ def remove_files(path):
         path (str): Path to remove *.egg-info and *.dist-info files.
     """
 
-    rmdirs = glob.glob(os.path.join(path, "*.egg-info")) + glob.glob(
-        os.path.join(path, "*.dist-info")
-    )
+    delete_list = ["*.egg-info", "*.dist-info"]
+    # ADDON-23349: Do not remove jsonschema dist-info/egg-info directory
```

Comment:
@mchavda-splunk can you please include details of why we don't want to remove jsonschema's dist-info folder, not referring to Jira ticket?

Reply:
Added reason and removed the Jira link